src/bunfig.zig

```diff
@@ -252,6 +253,26 @@ pub const Bunfig = struct {
                         this.ctx.test_options.coverage.enabled = expr.data.e_boolean.value;
                     }
 
+                    if (test_.get("coverageReporters")) |expr| {
+                        try this.expect(expr, .e_array);
+                        const items = expr.data.e_array.items.slice();
+                        const reporters = try this.allocator.alloc(TestCommand.Reporter, items.len);
+                        for (items, 0..) |item, i| {
+                            try this.expectString(item);
+                            if (TestCommand.ReporterMap.get(item.asString(bun.default_allocator) orelse "")) |reporter| {
```

Comment:
so this instead of using a map here, it could just be a simple chain of if else if else and `bun.strings.eqlComptime(x, "lcov")` etc

Reply:
Did so in https://github.com/oven-sh/bun/pull/11883/commits/19db90c71eb87ba7c7a998208c8bed28befc6065